exporters/trace/jaeger/src/test/java/io/opencensus/exporter/trace/jaeger/JaegerExporterHandlerIntegrationTest.java

```diff
@@ -46,219 +40,167 @@
 import io.opencensus.trace.Tracing;
 import io.opencensus.trace.samplers.Samplers;
 import java.io.IOException;
-import java.net.ConnectException;
-import java.net.SocketException;
 import java.util.Random;
-import java.util.logging.Level;
-import java.util.logging.Logger;
+import org.junit.Before;
+import org.junit.ClassRule;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
 
 public class JaegerExporterHandlerIntegrationTest {
-  private static final String JAEGER_HOST = "127.0.0.1";
-  private static final int OK = 200;
+  private static final String JAEGER_IMAGE = "jaegertracing/all-in-one:1.3";
+  private static final int JAEGER_HTTP_PORT = 16686;
+  private static final int JAEGER_HTTP_PORT_THRIFT = 14268;
   private static final String SERVICE_NAME = "test";
   private static final String SPAN_NAME = "my.org/ProcessVideo";
   private static final String START_PROCESSING_VIDEO = "Start processing video.";
   private static final String FINISHED_PROCESSING_VIDEO = "Finished processing video.";
 
   private static final Logger logger =
-      Logger.getLogger(JaegerExporterHandlerIntegrationTest.class.getName());
-  private static final Tracer tracer = Tracing.getTracer();
+      LoggerFactory.getLogger(JaegerExporterHandlerIntegrationTest.class);
 
   private HttpRequestFactory httpRequestFactory = new NetHttpTransport().createRequestFactory();
 
-  @Test(timeout = 30000)
-  public void exportToJaeger() throws InterruptedException, IOException {
-    assumeThat("docker is installed and running", isDockerInstalledAndRunning(), is(true));
-
-    final Process jaeger =
-        Runtime.getRuntime()
-            .exec(
-                "docker run --rm "
-                    + "-e COLLECTOR_ZIPKIN_HTTP_PORT=9411 -p5775:5775/udp -p6831:6831/udp "
-                    + "-p6832:6832/udp -p5778:5778 -p16686:16686 -p14268:14268 -p9411:9411 "
-                    + "jaegertracing/all-in-one:1.2.0");
-    waitForJaegerToStart(format("http://%s:16686", JAEGER_HOST));
```

Comment:
indents invalidated a lot of this as well changing the assertions. Trust me.. it is ok :P

Reply:
I was able to view the changes with --color-words and --ignore-all-space.